concepticondata/concepticon.tsv

```diff
@@ -3174,3 +3174,55 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3179	CORAL ROCK	The physical world	Limestone created from the skeleton of reef-forming stony corals (Scleractinia), displaying their growth patterns.	Person/Thing	
 3180	SHOOT (WITH SLINGSHOT)	Warfare and hunting	Use a rubber-based projectile weapon to propel projectiles against a target.	Action/Process	
 3181	MUSTARD	Food and drink	It can be a yellow sauce with a strong taste or a plant which is used to make mustard sauce.	Person/Thing	
+3182	MARRIAGE OR WEDDING	Kinship	The ceremony of marrying someone or the state of being married.	Person/Thing	
+3183	BREATH OR BREATHE	The body	The air that goes into one's lungs or the act of breathing	Other	
+3184	BE ALIVE OR LIFE	The body	The act of being alive or the life itself.	Other	
+3185	BE DEAD OR DIE OR KILL	The body	To be not alive or to cease to live or to cause the end of a life.	Other	
+3186	BE DEAD OR DEATH OR DIE	The body	To be not alive or the end of a life or to cease to live.	Other	
+3187	DROWNED	The body	To have perished by suffocation in water.	Property	
+3188	MIGHTY OR POWERFUL OR STRONG	Emotions and values	Holding much (political, social, ...) power or being done using a huge amount of energy or having great physical strength.	Property	
```

Comment:
This relates to the comments of POWERFUL above

Reply:
In this I included Key, List and Buck since they have all three words. I could re-link Key and List to STRONG since in Haspelmath the concept is named that.